src/Data/String.purs

```diff
@@ -210,10 +219,11 @@ foreign import _localeCompare
   -> String
   -> Ordering
 
--- | Replaces the first occurence of the first argument with the second argument.
+-- | Replaces the first occurence of the pattern with the replacement string.
+-- | * `replace (Pattern "http") (Replacement "https") "http://purescript.org"`
```

Comment:
Do you think it would work to include the result here?

Reply:
I will definitely lead to a line-wrap on Pursuit, but it's probably better to add it for consistency reasons.